<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailSenderApp</class>
 <widget class="QMainWindow" name="EmailSenderApp">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="theme_mail_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тема</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_mail_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текст письма</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QLabel" name="address_name_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес получателя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="theme_mail_text_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_name_text_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_mail_text_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="server_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сервер</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>yandex</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gmail</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mail</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="address_sender_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес отправителя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_sender_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pass_sender_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль отправителя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_sender_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="choose_label_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="list_view_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Список файлов</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_send_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
